releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,299 @@
+
+> *Screenshot here*
+
+:star2: *New types of events*: In course of implement the new Shipyard version in Keptn, the Keptn Cloud-events were streamlined and follow now the a common pattern. Basically, Keptn just sends out an event of type: `sh.keptn.event.{task.name}.triggered` and other services react on: 
+  * `sh.keptn.event.{task.name}.triggered`      > *sent out by Keptn*
+  * `sh.keptn.event.{task.name}.started`        > *sent out by a Keptn-service when starting the tasks*
+  * `sh.keptn.event.{task.name}.status.changed` > *sent out by a Keptn-service to inform about a status update*
+  * `sh.keptn.event.{task.name}.finished`       > *sent out by a Keptn-service when the task is finished*
+
+:sparkles: *Multi-cluster support*: Based on the implementation of Shipyard v0.2, Keptn - as a control-plane for delivery and remediation - is now capable of serving multiple clusters. This is known as the split between *Control plane* and *Execution plane*. For this use-case, the Keptn project offers to run the helm-service (to deploy) and jmeter-service (to test) on the *Execution plane*. This *Execution plane* can be on a cluster other than the cluster where Keptn is installed. 
+
+:dizzy: *Sequence screen in Keptn Bridge*: The new capabilities of Keptn for dealing with task sequences received a dedicated screen in the Keptn Bridge. This screen provides filtering capabilities and a stage-divided view of the performed delivery or remediation tasks. 
+
+> *Screenshot here*
+
+---
+
+**Supporting features:**
+
+:tada: *Query usage statistics of your Keptn*: With this release, it is possible to retrieve usage statistics of a Keptn by using the `/api/statistics/v1` endpoint. This returns the number of events processed in the specified time frame. 
+
+:star2: *Keptn CLI supports multiple Keptn installations*: The new Keptn CLI easies working with multiple Keptns since it recognizes switches between Kubernetes clusters and then asks for switching the context Keptn context too. Consequently, your CLI will be automatically connected to the Keptn running on another K8s cluster.   
+
+:star: *Deployment of custom Helm Charts*: An extension of the helm-service allows to deploy custom Helm Charts meaning that the Helm Chart can contain any custom resource and is not limited to a *Kubernetes service* and *deployment*. *Note:* When using this option, the automatic rollback capability of Keptn is not supported.
+
+:sparkles: *SLI breakdown displayed as a table in Keptn Bridge*: For the quality gates capabilities of Keptn, the SLI breakdown is now displayed as a table given a better overview of the individual results. 
+
+---
+
```

Comment:
Can we add a section here detailing end-user facing breaking changes (not neccessarily by GH issue), e.g.:

**API**

* Introduction of shipyard controller API via `/api/controlPlane/v1`
* Adding and updating resources now works via the following endpoitns: **TODO**
* Project Endpoints have been moved to: **TODO**
* Stage Endpoints have been moved to: **TODO**
* Service Endpoints have been moved to: **TODO**
* Evaluation (start-evaluation) Endpoint has been moved to: **TODO**
* Get-Events Endpoint has been moved to: **TODO**
* **TODO**: Please list all breaking changes for end-users/devs

**CLI**


- Evaluations are no longer triggered using `keptn send event start-evaluation`, but using `keptn trigger evaluation` (e.g., `keptn trigger evaluation --project=my-sockshop --service=foobar --stage=hardening`)
- Sending a configuration-change/new-artifact is no longer triggered using `keptn send event new-artifact`, but using `kept trigger delivery` (e.g., `keptn trigger delivery --project=sockshop --service=carts-db --image=docker.io/mongo --tag=4.2.2 --sequence=delivery-direct`)
- **TODO** Any more changes **important** and **breaking** changes? Please add them!

**BRIDGE**

- **TODO**: Add user-facing breaking changes